search.py

```diff
@@ -0,0 +1,23 @@
+# Name of this file: search.py
+# Location: at the root of the project
+# How to execute: python3 -m search.py query
+
+import sys
+import louis.db.api as api
+import louis.db as db
+from microbench import MicroBench
+import pandas as pd
+
+basic_bench = MicroBench()
+
+# Execute the SQL search function 
+@basic_bench
+def search(cursor, query):
+    api.search_from_text_query(cursor, query)
```

Comment:
t'as pas de return ici, c'est pour ça que tu as None

https://realpython.com/python-return-statement/#implicit-return-statements

A Python function will always have a return value. There is no notion of procedure or routine in Python. So, if you don’t explicitly use a return value in a return statement, or if you totally omit the return statement, then Python will implicitly return a default value for you. That default return value will always be None.

Reply:
even with the return it doesn't change the none I get, I guess it wasn't the only problem, I will keep digging into that